src/pages/Models/Details/ModelDetails.js

```diff
@@ -141,8 +139,38 @@ const generateUnitRows = modelStatusData => {
           { content: unit.workloadStatus.info }
         ]
       });
+
+      const subordinates = unit.subordinates;
+
+      if (subordinates) {
+        Object.keys(subordinates).forEach(key => {
```

Comment:
`Object.entries` might be a good method to use here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries